src/wsi/wsi_window.h

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+// The windows.h will define HWND/HMONITOR
+// to be types that it wants for the wsi we are compiling against.
+// ie. HWND     = their own window data ptr (eg. SDL_Window)
+//     HMONTIOR = their own monitor data ptr/a display index
+
+#include <windows.h>
+
+#include "wsi_mode.h"
+
+namespace dxvk::wsi {
+
+  /**
+    * \brief Impl-dependent state
+    */
+  struct DxvkWindowState {
+#ifdef DXVK_WSI_WIN32
```

Comment:
This whole struct should be in platform-specific headers.

Reply:
Agree.